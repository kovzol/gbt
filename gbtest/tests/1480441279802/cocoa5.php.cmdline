#!/bin/sh
cd tests/1480441279802
cat cocoa5.php.in | /usr/bin/timeout -k 120 60 /usr/bin/time /home/kovzol/workspace/cocoa-5.1/cocoa5 > cocoa5.php.out 2>cocoa5.php.err