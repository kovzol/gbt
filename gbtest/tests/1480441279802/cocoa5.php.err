0.11user 0.01system 0:00.13elapsed 96%CPU (0avgtext+0avgdata 21628maxresident)k
0inputs+8outputs (0major+4261minor)pagefaults 0swaps
