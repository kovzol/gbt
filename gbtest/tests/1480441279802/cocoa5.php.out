
Loading CoCoA packages from directory:
  /home/kovzol/workspace/cocoa-5.1/packages
   ______      ______      ___         ______
  / ____/___  / ____/___  /   |       / ____/
 / /   / __ `/ /   / __ `/ /| |______/___ `  
/ /___/ /_/ / /___/ /_/ / ___ /_____/___/ /  
`____/`____/`____/`____/_/  |_|    /_____/   
With CoCoALib and external libraries CDD FROBBY GFAN GSL NORMALIZ READLINE 
indent(VersionInfo()); -- for information about this version

# use QQ[x6,x5,x4,x3,x2,x1,x,y], DegRevLex;

# GBasis(ideal((x1),(x2),((((x2)-(1))*(x3))-((x1)*((x4)-(1)))),(x6),((0)+(((x5)<x2)-(1))*(x3))-((x1)*((x4)-(1)))),(x6),((0)+(((x5)^                         2)+(((x6)^2)+(((((x1)^2)+<-(1)))),(x6),((0)+(((x5)^2)+(((x6)^2)+(((((x1)^2)+(                         (x2)^2))-(((-(x3))*(-(x3)<)+(((x6)^2)+(((((x1)^2)+((x2)^2))-(((-(x3))*(-(x3))                         )+(((1)-(x4))*((1)-(x4)))<x2)^2))-(((-(x3))*(-(x3)))+(((1)-(x4))*((1)-(x4))))                         )+(((0)*((x5)*(x6)))+((((<+(((1)-(x4))*((1)-(x4)))))+(((0)*((x5)*(x6)))+((((2                         )*(-(x1)))*(x5))+(((2)*(-<+(((0)*((x5)*(x6)))+((((2)*(-(x1)))*(x5))+(((2)*(-(                         x2)))*(x6)))))))),((((x6)<*(-(x1)))*(x5))+(((2)*(-(x2)))*(x6)))))))),((((x6)-                         (x4))*((x)-(x3)))-(((x5)-<2)))*(x6)))))))),((((x6)-(x4))*((x)-(x3)))-(((x5)-(                         x3))*((y)-(x4)))),-2*x4^2<x4))*((x)-(x3)))-(((x5)-(x3))*((y)-(x4)))),-2*x4^2*                         x2+2*x4^2-2*x4*x3*x1+2*x4<3))*((y)-(x4)))),-2*x4^2*x2+2*x4^2-2*x4*x3*x1+2*x4*                         x2*y+2*x4*x2-2*x4*y-2*x4+<2+2*x4^2-2*x4*x3*x1+2*x4*x2*y+2*x4*x2-2*x4*y-2*x4+2                         *x3*x1*y+2*x2*x1*x5-2*x2*<2*y+2*x4*x2-2*x4*y-2*x4+2*x3*x1*y+2*x2*x1*x5-2*x2*x                         1*x-2*x2*x5^2+2*x2*x5*x-2<x3*x1*y+2*x2*x1*x5-2*x2*x1*x-2*x2*x5^2+2*x2*x5*x-2*                         x2*y-2*x1^2*x6+2*x1^2*y+2<*x-2*x2*x5^2+2*x2*x5*x-2*x2*y-2*x1^2*x6+2*x1^2*y+2*                         x1*x5*x6-2*x1*x5*y-2*x1*x<2*y-2*x1^2*x6+2*x1^2*y+2*x1*x5*x6-2*x1*x5*y-2*x1*x5                         +2*x1*x+2*x5^2-2*x5*x+2*y<1*x5*x6-2*x1*x5*y-2*x1*x5+2*x1*x+2*x5^2-2*x5*x+2*y)                         );
[x1, x2, x6, -x3, x5*x4 -x4*x -x5*y, 2*x5^2 -x5*x -x4*y +x4 +y -1, 2*x4^2 -x5*x -x4*y -3*x4 +y +1, -x5*x^2 -3*x4*x*y -2*x5*y^2 -x4*x +4*x5*y +x*y -2*x5 +3*x, -x4*x^2 +x4*y^2 +4*x5*x -3*x^2 -2*x4*y -y^2 +x4 +2*y -1, -x5*x*y -x4*y^2 -3*x5*x +2*x^2 +2*x4*y +y^2 -x4 -2*y +1, -x4*x*y^2 -x5*y^3 -x^3 -6*x4*x*y -x5*y^2 -x4*x +5*x5*y +4*x*y -3*x5 +4*x, -x^4 +x^2*y^2 -4*x4*y^3 +18*x^2*y +20*x4*y^2 +4*y^3 +32*x5*x -19*x^2 -28*x4*y -20*y^2 +12*x4 +28*y -12]

# 